guidelines.md

```diff
@@ -1,41 +1,76 @@
+
+For lightning talks (5 minutes), focus on sparking curiosity rather than covering every detail. If your talk has prerequisites, make sure the audience is aware upfront.
+
+### 2. Keep the Session Interactive
+Engagement is key. Make your presentation interactive by asking questions, encouraging participation, or conducting small activities throughout the talk. Polling tools like [slido.com](https://slido.com) can be used to gather real-time feedback or conduct polls (Not sponsored).
+
+#### A. Use Real-Life Examples
+Simplify complex ideas by using real-life examples or analogies. This makes technical concepts easier to understand and more relatable to the audience. You can also stick to one motivating example that you can refer to throughout your talk. This helps maintain audience engagement and creates a clear narrative that ties all your points together.
+
+#### B. Make the Presentation Visually Engaging
+Avoid text-heavy slides. Use visuals, diagrams, or videos to illustrate your points. Keep slide text to a minimum (4-5 lines max) and use images to support your message. Visual content helps convey information more effectively than blocks of text.
+
+#### C. Add Humour Thoughtfully
+A well-timed joke or light comment can lighten the mood and help you connect with the audience really fast. However, make sure that your humour is appropriate and relevant to the context.
+
+#### D. Having Code in Your Talk/Slides
+If done nicely, please include it, otherwise it could overwhelm the audience, and real-time tutorials in the talk are usually not recommended by many, but meetups are a safe space where you can try things and not get judged. We are very open to having people experiment with different formats and styles at our meetups.
+
+#### E. Include Hooks to Encourage Conversation
+Add points/hooks in your talk that will encourage the audience to approach you after the talk for further discussion. This not only adds value to your talk but also creates opportunities for networking and deeper engagement with attendees.
+
+### 3. Use the Stage to Your Advantage
+Engage with the audience by moving around the stage and making eye contact. Moving with purpose helps keep the energy high and reduces the risk of losing your audience’s attention.
+
+### 4. Vary Your Tone and Pitch
+Keep your delivery lively by varying your tone and pitch. A monotone voice can easily lose the audience’s attention, while vocal variation can highlight key points and keep the audience engaged.
+
+### 5. Highlight The Best Practices
+When discussing code or technical processes, especially when demonstrating live coding, emphasize best practices such as **security** and **ethical coding**. For instance, never display sensitive data, like API keys, during your presentation.
+
+---
+
+
+**Courtesy**: Anand B. Pillai, Aditi Juneja
+
+
+Happy contributing!
+
+---
+
+PS: Don't be too bothered by all of the above points and just be your true, authentic self and have fun :)
```

Comment:
Let's have them be bothered :smiling_imp: (this is a joke)

This sentence undermines the points made above and may give the reader a sense of bathos.

We can instead mention it in the beginning of the copy that these are not hard and fast rules but instead high-level guidelines.